### DISCUSSION TITLE
【委托】服务器招新

### DISCUSSION THREAD

**Discussion by Kunspring** [share121/inter-knot#4696] [委托]
![Image_1720799744524](https://github.com/user-attachments/assets/026fd0ca-7099-441a-8baf-f0bcf7e195c7)

腐竹很可爱，快来调戏（bushi
版本1.21.1java版支持fabric，群里有手机java启动器，服务器没装间歇泉
服务器以交友为主，游戏体验和腐竹反馈就好了（也不一定能解决
不玩服务器的话来群里聊聊天也可以鸭）
![qrcode_1725887559361](https://github.com/user-attachments/assets/abdf7919-cc5c-4d5a-9699-250528ef8c51)

**Comment by Kunspring** [comment]
![Image_1725887196846](https://github.com/user-attachments/assets/b08a0184-469d-4065-bafe-9bc10e742066)

**Reply by HollowInvestigativeAssociation** [reply]
我是萌新，带我！！！

**Comment by lllliufeng** [comment]
我来

**Comment by HollowInvestigativeAssociation** [comment]
萌新求带

**Comment by Tianyv0111** [comment]
萌新⚽️带

**Comment by Elysian222** [comment]
头像小老师？

**Reply by Kunspring** [reply]
对的！
![mobileqq_card_pic_1723051207323](https://github.com/user-attachments/assets/452ec828-f115-4cda-bd8d-d19345dc64c3)

**Comment by howtosky** [comment]
萌新求带<img src="https://bbs-static.miyoushe.com/static/2024/07/31/fdc5e0e30e577fb208f29effa93510ed_8442612635207222552.png" alt="云璃 期待" width="75" >

**Reply by Kunspring** [reply]
die die need（
